### ISSUE TITLE
PBJS debug console should show unsupported bidder errors from Prebid Server

### ISSUE THREAD

**Issue by kevinlisota** [prebid/Prebid.js#2234]
Opening in this repo per original issue [here](https://github.com/prebid/prebid-server/issues/385).

Prebid Server only supports a small subset of bidders compared to the client-side prebid.js. Prebid Server will respond to the auction request with an error message that says "unsupported bidder" when an unsupported request is sent.

When using PBJS debug console, this error does not display at all, yet seems a very important error. In the debug console, the only message was that bids were sent, and then the supported bidders returned bids.

In order to discover this error, I had to look in the browser console and inspect the response body that prebid server returned, which clearly shows the error.

To reproduce, make sure you have prebid server config up and running, like with AppNexus or elsewhere. Setup an ad request to a bidder that is NOT on the supported list: http://prebid.adnxs.com/pbs/v1/info/bidders. View the auction response in the browser console to see the error message.

To summarize, if prebid server responds with an "unsupported bidder" error message, that error message should be clearly displayed in the PBJS debug console.

**Comment by mkendall07** [comment]
Yep this makes sense
